LondonBritishLibrary/orient/BLorient12513.xml

```diff
@@ -0,0 +1,658 @@
+                                            <msItem xml:id="p2_i1.4.32.1">
+                                                <locus from="135vb"/>
+                                                <title ref="LIT3171Commen#Dersan32"/>
+                                            </msItem>
+                                            <msItem xml:id="p2_i1.4.32.2">
+                                                <locus from="137vb"/>
+                                                <title ref="LIT3171Commen#Tagsas32"/>
+                                            </msItem>
+                                        </msItem>
+                                        <msItem xml:id="p2_i1.4.33">
+                                            <locus from="139va"/>
+                                            <title ref="LIT3171Commen#Homily33"/>
+                                            <msItem xml:id="p2_i1.4.33.1">
+                                                <locus from="139va"/>
+                                                <title ref="LIT3171Commen#Dersan33"/>
+                                            </msItem>
+                                            <msItem xml:id="p2_i1.4.33.2">
+                                                <locus from="142va"/>
+                                                <title ref="LIT3171Commen#Tagsas33"/>
+                                            </msItem>
+                                        </msItem>
+                                        <msItem xml:id="p2_i1.4.34">
+                                            <locus from="143va"/>
+                                            <title ref="LIT3171Commen#Homily34"/>
+                                            <msItem xml:id="p2_i1.4.34.1">
+                                                <locus from="143va"/>
+                                                <title ref="LIT3171Commen#Dersan34"/>
+                                            </msItem>
+                                            <msItem xml:id="p2_i1.4.34.2">
+                                                <locus from="145vc"/>
+                                                <title ref="LIT3171Commen#Tagsas34"/>
+                                            </msItem>
+                                        </msItem>
+                                    </msItem>
+                                    <colophon xml:id="coloph1">
+                                        <locus target="#147rc"/>
+                                        <note>The date given according to the Era of the martyrs 
+                                            (<date type="translation" notBefore="1522" notAfter="1523" when-custom="1239" calendar="diocletian">1239</date>) 
+                                            is the only one, which is correct. The date according to the Era of the World (7005) must be corrected to 7015 and 
```

Comment:
To make the perception of the dates a little easier (esp. when they are equated to other dates), I usually write the indication CE after dates according to the common Gregorian calendar.

Reply:
There is no date in the Gregorian calendar except for those in notBefore / notAfter, that are not visible on the website and cannot indicated with CE inside the markup.

Reply:
Changed.

Reply:
I am confused. the CE (common era) date should be the Gregorian date, it cannot be the `when-custom` date which is according to this annotation a date in Diocletian calendar.

when-custom/notBefore-custom/notAfter-custom = alternative calendar
when/notBefore/notAfter = CE

Reply:
I am sorry for this confusion! Indeed the Common Era date is not mentioned in the note verbally. Probably, I should add it.